src/integrationTest/java/uk/gov/hmcts/reform/waworkflowapi/CamundaCreateTaskTest.java

```diff
@@ -168,8 +168,39 @@ public void overdueAndOverdueTaskIsNotCreated() {
         assertThat(processInstance).isEnded();
     }
```

Comment:
Will this test eventually move to wa-standalone-task-bpmn permanently? Or is the plan to have it here also?